electricwatch.py

```diff
@@ -0,0 +1,7 @@
+N=input('N = ')
+N=int(N)
```

Comment:
Для упрощения можно считывать данные и приводить их к типу _int_ в одной строчке кода: 
```suggestion
N=int(input('N = '))
```
В более сложных задачах с большим количеством входных параметров постарайся запрашивать у пользователя конкретные данные, а не абстрактные "N":
```suggestion
N=int(input('Введите N – количество секунд для перевода: '))
```